specs/DotNetProjectFile.Analyzers.Specs/Extensions/Microsoft.CodeAnalysis.Diagnostics.DiagnosticAnalyzer.cs

```diff
@@ -1,13 +1,54 @@
 using CodeAnalysis.TestTools.Contexts;
 using Specs.TestTools;
 using System.Collections.Concurrent;
+using System.Diagnostics.CodeAnalysis;
 using System.IO;
-using System.Threading;
+using System.Security.Cryptography;
 
 namespace Microsoft.CodeAnalysis.Diagnostics;
 
 internal static class DiagnosticAnalyzerExtensions
 {
+    [Pure]
+    public static ProjectAnalyzerVerifyContext ForInlineCsproj(
+        this DiagnosticAnalyzer analyzer,
+        [StringSyntax(StringSyntaxAttribute.Xml)] string content)
```

Comment:
I Like the use of `[StringSyntax(StringSyntaxAttribute.Xml)]` here.

Reply:
If only Visual Studio added actual support for the XML highlighting... But at least it should be future-proof if they ever add it